config/application.rb

```diff
@@ -42,5 +42,8 @@ class Application < Rails::Application
       Sequel::Model.db.extension :server_block
       Sequel::Model.db.extension :auto_literal_strings
     end
+
+    # Feature flags
+    config.include_excise_measure_units = ENV.fetch('INCLUDE_EXCISE_MEASURE_UNITS', 'false') == 'true'
```

Comment:
So the idiom in this app is to put these kinds of feature flags in the TradeTariffBackend module. I reckon just for consistency we should move this there?